src/assets/translations/en/main.json

```diff
@@ -967,5 +967,12 @@
     "to": "To",
     "for": "For",
     "transfers": "Transfers"
+  },
+  "plugins": {
+    "foxPage": {
+      "totalFoxValue": "Total Fox Value",
+      "foxToken": "ShapeShift FOX Token",
```

Comment:
Might be better to use `%{}` syntax for a dynamic asset symbol (but hardcode it for now)